chromebook-setup.sh

```diff
@@ -742,6 +742,10 @@ cmd_setup_fedora_rootfs()
     -e '/home/s/btrfs/ext4/' \
     -e 's/subvol=home,compress=zstd:1/defaults/' ./tmpdir/root/etc/fstab
 
+    # echo "Inserting Kernel Update script"
+    sudo chmod 777 96-chromebook.install
+    sudo cp 96-chromebook.install ./tmpdir/root/usr/lib/kernel/install.d/
```

Comment:
Instead of putting this file inside the source path, can we create a new directory called scripts and put the file there?

Reply:
cool, make sense